pkg/query-service/app/dashboards/provision.go

```diff
@@ -37,13 +37,17 @@ func readCurrentDir(dir string) error {
 			continue
 		}
 
-		_, apiErr := GetDashboard(data["uuid"].(string))
-		if apiErr == nil {
-			zap.S().Infof("Creating Dashboards: Error in file: %s\t%s", filename, "Dashboard already present in database")
-			continue
+		zap.S().Infof("Dashboard: file: %s\t uuid: %s", filename, data["uuid"].(string))
+
+		if dashboardUuid, ok := data["uuid"]; ok {
```

Comment:
If `data["uuid"]` doesn't exist in the map, I believe `data["uuid"].(string)` will panic, please check once.

Reply:
@srikanthccv  updated